packages/database/src/migrations/20220107061813-SetMapOverlayGroupCodeAsUniqueKey-modifies-schema.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  // 1. Delete duplicated map overlay group with code 'Root'.
+  //    Have checked it doesn't have any children in overlay relation table.
+  // 2. Set map overlay group code as unique key.
```

Comment:
Good work on removing this, I had no idea that there were 2 root mapOverlayGroups. I double checked and you are correct that the one with Id Root has no children so good riddance.

Reply:
Thanks @tcaiger, and yes it's a bit weird. Hope that doesn't break anything.